tests/virt_control_plane_test.go

```diff
@@ -175,29 +180,32 @@ var _ = Describe("[Serial][ref_id:2717][sig-compute]KubeVirt control plane resil
 					err = virtCli.CoreV1().Pods(flags.KubeVirtInstallNamespace).Evict(context.Background(), &v1beta1.Eviction{ObjectMeta: metav1.ObjectMeta{Name: pod.Name}})
 					// trying to evict the last running pod in this list should fail
 					if index == len(runningPods)-1 {
-						Expect(err).To(HaveOccurred(), "no error occurred on evict of last pod")
+						if replicas > 1 {
+							Expect(err).To(HaveOccurred(), "no error occurred on evict of last pod")
+						} else {
+							Expect(err).ToNot(HaveOccurred(), "error occurred on evict of single-replica pod")
+						}
 					} else {
 						Expect(err).ToNot(HaveOccurred())
 					}
 				}
 			}
 
-			It("[test_id:2830]last eviction should fail for virt-controller pods", func() { test("virt-controller") })
-			It("[test_id:2799]last eviction should fail for virt-api pods", func() { test("virt-api") })
+			It("[test_id:2830]last eviction should fail for multi-replica virt-controller pods", func() { test("virt-controller") })
```

Comment:
What would you say about following `last eviction should fail for multi-replica virt-controller pods` -> `last eviction should fail **only** for multi-replica virt-controller pods`